fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -309,6 +311,9 @@
     @VariableMgr.VarAttr(name = PIPELINE_DOP)
     private int pipelineDop = 0;
 
+    @VariableMgr.VarAttr(name = PIPELINE_MAX_ROW_COUNT_PER_SCAN_OPERATOR)
+    private long pipelineMaxRowCountPerScanOperator = 262144;
```

Comment:
This variable could be invisible for users. `flag = VariableMgr.INVISIBLE`.

Why max row count per san operator so small?

Reply:
this variable should be assigned an robust and  emprical value, here the value is quite heuristic(that is to say, inaccurate), a better suggestion is welcome.

Reply:
This variable could be invisible for users. flag = VariableMgr.INVISIBLE. We shouldn't  expect the user change the value.

Reply:
ok